gravitee-apim-gateway/gravitee-apim-gateway-reactor/src/main/java/io/gravitee/gateway/reactor/handler/Acceptor.java

```diff
@@ -15,14 +15,10 @@
  */
 package io.gravitee.gateway.reactor.handler;
 
-import io.gravitee.definition.model.ExecutionMode;
-
 /**
- * This class represents a http acceptor link to a {@link ReactorHandler}.
- *
  * @author David BRASSELY (david.brassely at graviteesource.com)
  * @author GraviteeSource Team
  */
-public interface HttpAcceptorHandler extends HttpAcceptor {
-    <T extends ReactorHandler> T target();
+public interface Acceptor<T extends Acceptor<T>> extends Comparable<T> {
```

Comment:
Acceptor<T extends Acceptor<T>> : I am sorry but I don't get this one. I don't see T used in the interface, should we remove it?

Reply:
The `T` is used for the` Comparable<T>`

Reply:
Could it be  `public interface Acceptor<T> extends Comparable<Acceptor<T>>` ?